Src/Plugins/Security/HelpersUtilitiesAndExtensions/DomainFilteringHelper.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.HelpersUtilitiesAndExtensions
+{
+    internal static class DomainFilteringHelper
+    {
+        public static readonly HashSet<string> LocalhostList = new HashSet<string>(StringComparer.OrdinalIgnoreCase)
+        {
+            "localhost",
+            "(local)",
+            "127.0.0.1",
+        };
+
+        public static string StandardizeLocalhostName(string hostName)
+        {
+            if (LocalhostList.Contains(hostName))
+            {
+                return "localhost";
+            }
+
+            return hostName;
+        }
+
+        public static string HostExclusion(ref string host, IEnumerable<string> hostList = null)
```

Comment:
>ref string host [](start = 43, length = 15)

here u are using ref string and above only string. should we change this to string host only?

Reply:
Done.

---
In reply to: [575580124](https://github.com/microsoft/sarif-pattern-matcher/pull/189#discussion_r575580124) [](ancestors = 575580124)